DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/DelegateCourses/SearchableDelegateCourseStatisticsViewModel.cs

```diff
@@ -15,6 +15,7 @@ public SearchableDelegateCourseStatisticsViewModel(CourseStatisticsWithAdminFiel
             CustomisationId = courseStatistics.CustomisationId;
             DelegateCount = courseStatistics.DelegateCount;
             InProgressCount = courseStatistics.InProgressCount;
+            CompletedCount = courseStatistics.CompletedCount;

```

Comment:
I think this takes into account Removed records, which the ticket says it shouldn't.

Reply:
Good catch, but I won't be fixing this since we've decided (see the slack thread @SteveJacksonSoft linked) to include removed progress records. Turned out to be a lucky error.